src/Resources/contao/classes/DataContainer.php

```diff
@@ -902,8 +902,9 @@ protected function initPicker()
 
 		$this->setPickerValue();
 
-		$strDriver = 'DC_' . $GLOBALS['TL_DCA'][$this->strPickerTable]['config']['dataContainer'];
-		$objDca = new $strDriver($this->strPickerTable);
+		// Create a DataContainer object (see #886)
+		$objDca = new \DataContainer();
```

Comment:
Unfortunately, the `onload_callback` does not only check permissions, it might also alter the DCA, add new fields etc. that are important when listing or editing the records.